banks-client/src/lib.rs

```diff
@@ -525,6 +525,7 @@ mod tests {
     }
 
     #[test]
+    #[allow(clippy::result_large_err)]
```

Comment:
interesting that this lint missed all the instances where we return `Result<(), BankClientError>` via `Future` :thinking: